src/acom_music_box/music_box.py

```diff
@@ -246,12 +226,13 @@ def solve(self, output_path=None, callback=None):
 
         return df
 
-    def readConditionsFromJson(self, path_to_json):
+    def loadJson(self, path_to_json, box_model):
```

Comment:
```suggestion
    def loadJson(self, path_to_json):
```

`box_model` is the same as `self`. `self` in python is equivalent to `this` in c++. See the usage below